### PR TITLE
Add <Nav /> Component

### PR DESCRIPTION
**Pull request by jmunozz**
# FRONTEND  

- Create a smart `<Nav />` component that wraps logic for modifying nickname.
- All nicknames changes are made through a modal dialog.
- Add new actions `UPDATE_NICKNAMES` and 2 actions related to the modal itself.
- Add new reducer functions for nickname and for the modal itself.

**todo**
_If player has already joined a game, nickname change must be notified to other players through the game object._

### CODE REVIEW

File: src/server/config/routes.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ const INDEX = path.resolve(__dirname, '../../client', 'index.html');
  * @returns {void}
  */
 export default (app) => {
-  app.get('/games', (req, res) => {
-    const gamesWaiting = getGames().filter(game => (!game.hasStarted));
+  app.get('/api/games', (req, res) => {
+    const gamesWaiting = allGames.filter(game => (!game.hasStarted));
```
Comment (thifranc) [inline src/server/config/routes.js]: allGames is deprecated, use getGames from handlers/games now

File: src/client/actions/tetris.js [hunk 2]
```diff
@@ -163,28 +164,27 @@ export function move(event) {
...
 }
 function movePieceDown(dispatch, getState) {
   console.log('movePieceDown');
-  drawWithNextPiece(dispatch, getState, (currentPiece) => ({...currentPiece, ...{ x: currentPiece.x + 1 }}));
+  drawWithNextPiece(dispatch, getState, currentPiece => ({ ...currentPiece, ...{ x: currentPiece.x + 1 } }));
 }
 // Will move piece to the lowest posible position.
 function stickPieceDown(dispatch, getState) {
-  console.log("stickPieceDown");
+  console.log('stickPieceDown');
```
Comment (thifranc) [inline src/client/actions/tetris.js:183]: nice eslint !

**Review by thifranc — COMMENTED** [review]
nice eslint
change components names as their not really explicit imo
e.g. : AllGames should be gamesList or Games
same for playersList
also why does gameSettings contains players, that would be separated I'd say as players aren't really a setting
fix conflicts and all good

File: src/client/actions/index.js [hunk 3]
```diff
@@ -2,10 +2,12 @@ import * as actionsConnexion from './connexion';
...
 import * as actionsSocket from './socket';
 import * as actionsGame from './game';
+import * as actionPlayer from './player';
 
 export default {
   ...actionsTetris,
   ...actionsConnexion,
   ...actionsSocket,
-  ...actionsGame
+  ...actionPlayer,
```
Comment (atoulous) [inline src/client/actions/index.js]: actionS

**Review by thifranc — COMMENTED** [review]
Awesome !

**Review by atoulous — CHANGES_REQUESTED** [review]
just rename actionPlayer to actionsPlayer

**Pull request merged**